ext/parquet/parquet.go

```diff
@@ -37,8 +40,40 @@ func (f *Format) NewDecoder(r io.Reader) (feedx.FormatDecoder, error) {
 }
 
 // NewEncoder implements Format.
-func (*Format) NewEncoder(w io.Writer) (feedx.FormatEncoder, error) {
-	return nil, fmt.Errorf("not implemented")
+func (f *Format) NewEncoder(w io.Writer) (feedx.FormatEncoder, error) {
```

Comment:
so no, the idea was to rewrite decoder using the new lib. the encoder won't work that way. we need to add a new `Schema` field to `type Format` and it MUST be present to create encoders. but again, let's re-implement decoder first and then add Encoder next